package.json

```
@@ -1,10 +1,11 @@
 {
   "name": "colman-webapplications",
   "version": "1.0.0",
-  "main": "app.js",
+  "main": "app.ts",
+  "type": "module",
   "scripts": {
-    "start": "node app.js",
-    "devstart": "nodemon app.js",
+    "start": "node app.ts",
```

```suggestion
    "start": "tsc && node app.js",
```
`node` can't run typescript files.